example/CMakeLists.txt

```diff
@@ -33,17 +33,19 @@ function(add_example target)
     target_link_libraries(${target} PRIVATE ${ARGN})
 endfunction()
 
-add_example(conversion_factor mp-units::core-fmt mp-units::core-io mp-units::si)
-add_example(custom_systems mp-units::core-io mp-units::si)
-add_example(hello_units mp-units::core-fmt mp-units::core-io mp-units::si mp-units::si-international)
-add_example(measurement mp-units::core-io mp-units::si)
-add_example(si_constants mp-units::core-fmt mp-units::si)
+add_example(v2_framework mp-units::core-fmt mp-units::core-io mp-units::si)
 
-if(NOT ${projectPrefix}LIBCXX)
-    add_subdirectory(glide_computer)
-endif()
+# add_example(conversion_factor mp-units::core-fmt mp-units::core-io mp-units::si)
+# add_example(custom_systems mp-units::core-io mp-units::si)
+# add_example(hello_units mp-units::core-fmt mp-units::core-io mp-units::si mp-units::si-international)
+# add_example(measurement mp-units::core-io mp-units::si)
+# add_example(si_constants mp-units::core-fmt mp-units::si)
 
-add_subdirectory(aliases)
-add_subdirectory(kalman_filter)
-add_subdirectory(literals)
-add_subdirectory(references)
+# if(NOT ${projectPrefix}LIBCXX)
+# add_subdirectory(glide_computer)
```

Comment:
I think this is a very valuable example in determining at a glance the success story of the "V2 framework". Rather than excluding it completely, I'd prefer if exclusions were done in the code.

Reply:
I will be enabling the files one by one when they will become supported. I do not intend to delete anything besides unused framework files (I did remove some already).